easybuild/easyconfigs/r/RNAcode/RNAcode-0.3-foss-2017a.eb

```diff
@@ -0,0 +1,26 @@
+
+# http://github.com/downloads/wash/rnacode/RNAcode-0.3.tar.gz
+easyblock = 'ConfigureMake'
+
+name = 'RNAcode'
+version = '0.3'
+
+homepage = 'https://wash.github.io/rnacode/'
+description = """RNAcode - Analyze the protein coding potential in multiple sequence alignments"""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = ['http://github.com/downloads/wash/rnacode/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['86c600a07ecd04d243fd6eb3c6cc5b20fd6159f34f69423e7391c0ce8f311048']
+
+dependencies = [
+]
```

Comment:
@ebirn - if there aren't any dependencies, you can simply remove this section completely. Please do so.